tree/ntuple/v7/src/RNTupleDescriptor.cxx

```diff
@@ -33,16 +33,10 @@
 
 bool ROOT::Experimental::RFieldDescriptor::operator==(const RFieldDescriptor &other) const
 {
-   return fFieldId == other.fFieldId &&
-          fFieldVersion == other.fFieldVersion &&
-          fTypeVersion == other.fTypeVersion &&
-          fFieldName == other.fFieldName &&
-          fFieldDescription == other.fFieldDescription &&
-          fTypeName == other.fTypeName &&
-          fNRepetitions == other.fNRepetitions &&
-          fStructure == other.fStructure &&
-          fParentId == other.fParentId &&
-          fLinkIds == other.fLinkIds;
+   return fFieldId == other.fFieldId && fFieldVersion == other.fFieldVersion && fTypeVersion == other.fTypeVersion &&
```

Comment:
The previous line splitting is a bit more readable, IMHO. (and I can not tell whether this change is white space only or has a function change too).

Reply:
I do agree but `clang-format` thinks otherwise.

Reply:
Please consider using:
```
// clang-format on
// clang-format off
```
here ....